addons/web/static/src/legacy/scss/kanban_examples_dialog.scss

```diff
@@ -67,31 +63,17 @@
 
 // kanban ghost card
 .o_kanban_examples {
-    display: flex;
-    padding: 0 6px;
-
     .o_kanban_examples_group {
-        flex: 1 1 100%;
-        margin: 10px;
+        flex-basis: 100%;
 
         .o_kanban_examples_ghost {
-            padding: 5px;
-            margin-top: -1px;
-            border: thin solid map-get($grays, '300');
-            background-color: white;
-            .o_ghost_content {
-                height: 12px;
-                background: map-get($grays, '300');
-                &.o_ghost_tag {
-                    display: inline-block;
-                    width: 40%;
-                    margin-top: 10px;
-                }
+            &.o_collapse {
+                margin-top: -1px;
             }
+
             .o_ghost_avatar {
                 height: 20px;
                 width: 20px;
```

Comment:
we're waiting for the avatars PR I guess ;)

Reply:
Indeed, this PR could simplify this part 👍 .